bucket/godot-beta.json

```diff
@@ -0,0 +1,36 @@
+{
+    "version": "4.0-beta1",
+    "description": "A feature-packed, cross-platform game engine to create 2D and 3D games from a unified interface.",
+    "homepage": "https://godotengine.org/",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://downloads.tuxfamily.org/godotengine/4.0/beta1/Godot_v4.0-beta1_win64.exe.zip"
+        },
+        "32bit": {
+            "url": "https://downloads.tuxfamily.org/godotengine/4.0/beta1/Godot_v4.0-beta1_win32.exe.zip"
+        }
+    },
+    "pre_install": "Get-ChildItem \"$dir\\Godot_*.exe\" | Rename-Item -NewName \"$dir\\godot.exe\"",
+    "bin": "godot.exe",
+    "shortcuts": [
+        [
+            "godot.exe",
+            "Godot Engine"
+        ]
+    ],
+    "checkver": {
+        "url": "https://downloads.tuxfamily.org/godotengine/$majorVersion.$minorVersion/",
+        "regex": "beta(?<beta>[\\d.]+)"
```

Comment:
Sadly this checkver won't work, you can't use $majorVersion or $minorVersion here. You'll have to write a checkver script to navigate the download directory (https://downloads.tuxfamily.org/godotengine/), open the latest version number folder, and then run this regex for the latest beta version. Would you like to have a go at writing it or would you rather I do it?

Reply:
Hi @tech189, thank you for your review

I have no idea how to do this script, if you can do it, I would be happy to :)